lanelet2_io/src/OsmHandlerLoad.cpp

```diff
@@ -408,7 +413,12 @@ void registerIds(const MapT& map) {
 }
 
 void testAndPrintLocaleWarning(ErrorMessages& errors) {
+  struct lconv * lc = localeconv();
```

Comment:
I think you can get rid of this change by just unconditionally including `<clocale>`, this include seems to be missing too.